### DISCUSSION TITLE
김미진 - 의료데이터 - cancer inhibitor 분석

### DISCUSSION THREAD

**Discussion by alwls5773** [dinnercoding/wanted-pre-onboarding-challenge-data#106] [2023년 11월 프리온보딩 데이터 챌린지]
**도메인 선정 이유**
- 주전공이 생명이고, 생명 분야에서 암 관련 데이터를 분석하고자 선정했습니다.

**구체적인 문제 정의**
- protein kinase의 경우 IC50이 10uM보다 낮으면 inhibitor라고 간주됩니다. 주요 kinase를 분석하여 어떤 inhibior가 암을 치료하는데 도움이 되는지 찾고자 합니다.
- 더 나아가 예측 모델을 만들어 보고자 합니다.

**데이터 업로드**
https://www.kaggle.com/datasets/xiaotawkaggle/inhibitors

**Comment by ghost** [comment]
미진님! 본인의 전공을 살려서 암을 치료하는데 도움이 되는 주요한 kinase를 분석하신다니 인상적이네요. 예측 모델까지 잘 만드시길 기대하겠습니다. 

다만, 비전공자이신 분들은 protein kinase, IC50 등의 용어를 낯설어하시기 때문에 용어 설명에 대한 부분이 가미되면 독자의 인지 부담을 줄여주기 때문에 훨씬 좋을 것 같아요. 물론, 어느 정도의 배경지식을 알아야 이해할 수 있는 전문적인 내용도 있을 텐데요. 회사에서 협업하다보면 생각보다 도메인에 대한 지식을 잘 모르시는 분들에게 설명을 해야할 때도 종종 있습니다. 그럴 때를 대비해서 독자들의 인지부하를 낮추는 연습을 해두시면 도움이 되실 거라 생각합니다 :)
